# EDA ANALYSIS - VICTOR HORCAS 


#### Análisis exploratorio de datos sobre un dataset de los mejores atletas de crossfit a nivel mundial. 

En este notebook realizaré la visualización de los datos de forma general, el estudio del dataset y las transformaciones y limpiezas necesarias.


#### Importamos todas las librerías que vamos a utilizar en este proyecto.

In [1]:
import seaborn as sns
import matplotlib.pyplot as plt
import pandas as pd
import numpy as np

## 1. Exploración de los datos

#### Cargamos el dataset escogido para el análisis.

In [ ]:
import kagglehub

In [ ]:
crossfit = kagglehub.dataset_download("tgomesjuliana/crossfit-competitions")
crossfit

In [2]:
crossfit_df = pd.read_csv("D:\\firstprojecteda\\data\\consolidated_athletes_information.csv")

#### Previsualización de forma general de los datos

In [5]:
# Base de datos entera
crossfit_df

competitorId       competitorName firstName     lastName gender  \
0           469656.0        Jeffrey Adler   Jeffrey        Adler      M   
1            34796.0        Scott Panchik     Scott      Panchik      M   
2           105875.0          Travis Mead    Travis         Mead      M   
3           310970.0        Saxon Panchik     Saxon      Panchik      M   
4            11435.0  Richard Froning Jr.   Richard  Froning Jr.      M   
...              ...                  ...       ...          ...    ...   
399337      872275.0     Caroline Stanley  Caroline      Stanley      F   
399338      174547.0      Amanda Barnhart    Amanda     Barnhart      F   
399339      121033.0         Sydney Wells    Sydney        Wells      F   
399340      503582.0        Alexis Raptis    Alexis       Raptis      F   
399341     1547449.0            Emma Cary      Emma         Cary      F   

       countryOfOriginName     regionName  affiliateId  \
0                   Canada  North America      18059.0   
1            United States  North America       7991.0   
2            United States  North America       9155.0   
3            United States  North America      22505.0   
4            United States  North America       3220.0   
...                    ...            ...          ...   
399337       United States  North America       3617.0   
399338       United States  North America      14583.0   
399339       United States  North America      19593.0   
399340       United States  North America      25335.0   
399341       United States  North America       1893.0   

                  affiliateName   age height  weight  overallRank  \
0           CrossFit Wonderland  27.0  69 in  197 lb          1.0   
1            CrossFit Mentality  33.0  69 in  187 lb          2.0   
2          Iron Valley CrossFit  34.0  73 in  205 lb          3.0   
3            CrossFit Cliffside  25.0  69 in  180 lb          4.0   
4               CrossFit Mayhem  33.0  69 in  194 lb          5.0   
...                         ...   ...    ...     ...          ...   
399337    College Hill CrossFit  23.0  63 in  145 lb          NaN   
399338       CrossFit High Gear  31.0  67 in  160 lb          NaN   
399339  CrossFit East Nashville  28.0  66 in  151 lb          NaN   
399340             TTT CrossFit  24.0  65 in  155 lb          NaN   
399341          CrossFit Naples  19.0  64 in  140 lb          NaN   

        overallScore  year competition  height_cm  weight_kg  
0              101.0  2021        open      176.0       90.0  
1              141.0  2021        open      176.0       85.0  
2              165.0  2021        open      186.0       93.0  
3              217.0  2021        open      176.0       82.0  
4              254.0  2021        open      176.0       88.0  
...              ...   ...         ...        ...        ...  
399337           0.0  2023       games      161.0       66.0  
399338           0.0  2023       games      171.0       73.0  
399339           0.0  2023       games      168.0       69.0  
399340           0.0  2023       games      166.0       71.0  
399341           0.0  2023       games      163.0       64.0  

[399342 rows x 18 columns]

In [6]:
# 5 primeras filas
crossfit_df.head (5)

competitorId       competitorName firstName     lastName gender  \
0      469656.0        Jeffrey Adler   Jeffrey        Adler      M   
1       34796.0        Scott Panchik     Scott      Panchik      M   
2      105875.0          Travis Mead    Travis         Mead      M   
3      310970.0        Saxon Panchik     Saxon      Panchik      M   
4       11435.0  Richard Froning Jr.   Richard  Froning Jr.      M   

  countryOfOriginName     regionName  affiliateId         affiliateName   age  \
0              Canada  North America      18059.0   CrossFit Wonderland  27.0   
1       United States  North America       7991.0    CrossFit Mentality  33.0   
2       United States  North America       9155.0  Iron Valley CrossFit  34.0   
3       United States  North America      22505.0    CrossFit Cliffside  25.0   
4       United States  North America       3220.0       CrossFit Mayhem  33.0   

  height  weight  overallRank  overallScore  year competition  height_cm  \
0  69 in  197 lb          1.0         101.0  2021        open      176.0   
1  69 in  187 lb          2.0         141.0  2021        open      176.0   
2  73 in  205 lb          3.0         165.0  2021        open      186.0   
3  69 in  180 lb          4.0         217.0  2021        open      176.0   
4  69 in  194 lb          5.0         254.0  2021        open      176.0   

   weight_kg  
0       90.0  
1       85.0  
2       93.0  
3       82.0  
4       88.0

Visualizamos información básica de nuestro dataset y los tipos de datos que contiene.

In [7]:
crossfit_df.describe()

competitorId    affiliateId            age    overallRank  \
count  3.993420e+05  343166.000000  399342.000000  399262.000000   
mean   1.249626e+06   13407.674816      35.036841   55504.236694   
std    7.227477e+05    8562.634020       8.101685   42532.260165   
min    8.600000e+01       4.000000      16.000000      -1.000000   
25%    6.559250e+05    5550.000000      29.000000   17732.000000   
50%    1.239003e+06   13295.000000      34.000000   48394.000000   
75%    1.942362e+06   20161.000000      41.000000   87406.750000   
max    2.523369e+06   32333.000000      54.000000  157742.000000   

        overallScore           year      height_cm      weight_kg  
count  399342.000000  399342.000000  399342.000000  399342.000000  
mean   203432.557287    2022.035338     173.984727      77.803296  
std    143953.597002       0.820031       9.333170      13.075751  
min         0.000000    2021.000000     150.000000      50.000000  
25%     79336.000000    2021.000000     168.000000      68.000000  
50%    186778.000000    2022.000000     175.000000      79.000000  
75%    307736.000000    2023.000000     181.000000      87.000000  
max    582777.000000    2023.000000     200.000000     110.000000

Haciendo uso de la función "describe" podemos ver algunos datos relevantes para nuestro análisis como puede ser la media de edad (unos 35 años), el período de años mínimo y máximo en el que está basado este dataset (2021 - 2023), la altura (174 cm) y peso medios (78 kg) y algunos datos más específicos como son los percentiles 25, 50 y 75 que se utilizan para indicar la posición de un valor en relación con el resto de los datos.

In [8]:
crossfit_df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 399342 entries, 0 to 399341
Data columns (total 18 columns):
 #   Column               Non-Null Count   Dtype  
---  ------               --------------   -----  
 0   competitorId         399342 non-null  float64
 1   competitorName       399340 non-null  object 
 2   firstName            399341 non-null  object 
 3   lastName             399338 non-null  object 
 4   gender               399342 non-null  object 
 5   countryOfOriginName  396587 non-null  object 
 6   regionName           399342 non-null  object 
 7   affiliateId          343166 non-null  float64
 8   affiliateName        343166 non-null  object 
 9   age                  399342 non-null  float64
 10  height               399342 non-null  object 
 11  weight               399342 non-null  object 
 12  overallRank          399262 non-null  float64
 13  overallScore         399342 non-null  float64
 14  year                 399342 non-null  int64  
 15  competition      

In [9]:
crossfit_df.dtypes

competitorId           float64
competitorName          object
firstName               object
lastName                object
gender                  object
countryOfOriginName     object
regionName              object
affiliateId            float64
affiliateName           object
age                    float64
height                  object
weight                  object
overallRank            float64
overallScore           float64
year                     int64
competition             object
height_cm              float64
weight_kg              float64
dtype: object

Con estas dos funciones obtenemos un recuento de valores no nulos y vemos el tipo de datos que tenemos (float, object e integer).

In [10]:
crossfit_df.columns

Index(['competitorId', 'competitorName', 'firstName', 'lastName', 'gender',
       'countryOfOriginName', 'regionName', 'affiliateId', 'affiliateName',
       'age', 'height', 'weight', 'overallRank', 'overallScore', 'year',
       'competition', 'height_cm', 'weight_kg'],
      dtype='object')

Aquí tenemos de un vistazo rápido todas las columnas (variables) que posee nuestro dataset.

Podemos ver que existen dos variables para la altura y dos para el peso (se considera eliminar una de cada una).

In [8]:
crossfit_df[["countryOfOriginName"]].value_counts()

countryOfOriginName
United States          191436
France                  22338
United Kingdom          21650
Australia               20420
Canada                  17168
                        ...  
Djibouti                    1
Malawi                      1
Gambia                      1
Togo                        1
Tuvalu                      1
Name: count, Length: 175, dtype: int64

In [11]:
crossfit_df[["competition"]].value_counts()

competition  
open             369450
quarterfinals     27943
semifinals         1709
games               240
Name: count, dtype: int64

Considero que todas las variables de este análisis son intuitivas y faciles de entender excepto la variable "competition" que puede no entenderse:

open --> Pruebas o ejercicos que puede realizar todo el mundo (de forma gratuita y sin inscribirse en ninguna competición aunque en parte lo sea). Normalmente la gente graba su rendimiento en estas pruebas y sirven como clasificatorio para pruebas superiores.

games --> Es la competición de crossfit más grande a nivel mundial, a esta competición sólo llegan los mejores atletas tras pasar varias pruebas y tener buenas marcas en una serie de competiciones previas.

#### VALORES NULOS 

In [12]:
# Sumamos los valores nulos por cada columna
crossfit_df.isna().sum() 

competitorId               0
competitorName             2
firstName                  1
lastName                   4
gender                     0
countryOfOriginName     2755
regionName                 0
affiliateId            56176
affiliateName          56176
age                        0
height                     0
weight                     0
overallRank               80
overallScore               0
year                       0
competition                0
height_cm                  0
weight_kg                  0
dtype: int64

In [3]:
# Calculo el porcentaje de valores nulos por columna para considerar si elimino la columna completa o trato esos valores nulos
porcentaje_nulos = crossfit_df.isna().mean() * 100
print(porcentaje_nulos)

competitorId            0.000000
competitorName          0.000501
firstName               0.000250
lastName                0.001002
gender                  0.000000
countryOfOriginName     0.689885
regionName              0.000000
affiliateId            14.067140
affiliateName          14.067140
age                     0.000000
height                  0.000000
weight                  0.000000
overallRank             0.020033
overallScore            0.000000
year                    0.000000
competition             0.000000
height_cm               0.000000
weight_kg               0.000000
dtype: float64


El dataset escogido no cuenta con gran cantidad de datos nulos, en especial, en las columnas más necesarias para nuestro proyecto como son el "competitorId", "gender", "age" y "overallRank". Las columnas con más valores nulos se podría considerar eliminarlas, ya que, aportan identificadores poco relevantes para nuestro proyecto.

In [10]:
df_games = crossfit_df[(crossfit_df["competition"] == "games") & (crossfit_df["year"] == 2023)]
df_games[["competitorName", "gender", "overallRank", "overallScore"]].sample(10)

competitorName gender  overallRank  overallScore
399310      Matilde Garnes      F          NaN           0.0
399267  Henrik Haapalainen      M          NaN           0.0
399274         Jelle Hoste      M          NaN           0.0
399303  Annie Thorisdottir      F          NaN           0.0
399305       Karin Freyová      F          NaN           0.0
399330      Bethany Flores      F          NaN           0.0
399297      Alex Vigneault      M          NaN           0.0
399338     Amanda Barnhart      F          NaN           0.0
399320    Danielle Brandon      F          NaN           0.0
399289      Colten Mertens      M          NaN           0.0

Para el año 2023 esta base de datos no tiene información sobre el rendimiento de los atletas en la competición "games". Tiene los puestos del ranking con valores nulos y los valores de "score" con valores igual a 0. Por tanto no podremos realizar un análisis de esta competición para este año en concreto.

#### VALORES DUPLICADOS 

Para mi análisis me interesa que para cada año no se duplique la variable "competitor_id", ya que, eso significa que está teniendo en cuenta a varios atletas con la misma identificaión en un mismo año.

In [4]:
# Cuenta el número de competiciones distintas por año y atleta
competiciones_por_atleta = crossfit_df.groupby(["year", "competitorId"])["competition"].nunique().reset_index()

# Filtra atletas con más de una competición en el mismo año
repetidos = competiciones_por_atleta[competiciones_por_atleta["competition"] > 1]
repetidos

year  competitorId  competition
13      2021        1653.0            2
21      2021        1690.0            4
27      2021        1718.0            2
31      2021        1758.0            2
37      2021        1775.0            2
...      ...           ...          ...
369276  2023     2512006.0            2
369345  2023     2513005.0            2
369390  2023     2513793.0            2
369408  2023     2514267.0            2
369538  2023     2516349.0            2

[27892 rows x 3 columns]

Este dataset cuenta con registros de varias competiciones para un mismo atleta dentro de un mismo año. Por tanto tiene sentido que existan valores duplicados para un mismo atleta dentro de un mismo año. Tendría que considerar los duplicados que fueran filas exactamente iguales unas a otras.

#### VALORES ÚNICOS 

In [6]:
crossfit_df[["gender"]].value_counts()

gender
M         263902
F         135440
Name: count, dtype: int64

Para la variable género nos interesa que sólo existan dos valores únicos, lo cuál si que ocurre en esta ocasión.

M --> Male (Masculino)

F --> Female (Femenino)

In [7]:
crossfit_df[["year"]].value_counts()

year
2023    141574
2022    130306
2021    127462
Name: count, dtype: int64

In [17]:
print(crossfit_df["year"].unique())

[2021 2022 2023]


Para la variable año sólo existen tres valores únicos, ya que, esta base de datos recopila valores entre el año 2021 - 2023.

#### VALORES ATÍPICOS 

In [18]:
print(crossfit_df[crossfit_df["overallScore"] < 0])

Empty DataFrame
Columns: [competitorId, competitorName, firstName, lastName, gender, countryOfOriginName, regionName, affiliateId, affiliateName, age, height, weight, overallRank, overallScore, year, competition, height_cm, weight_kg]
Index: []


La variable "overallScore" no presenta valores atípicos.

In [5]:
crossfit_df[crossfit_df["overallRank"] < 0]

competitorId    competitorName firstName   lastName gender  \
257361      458446.0  Nasser Alruwayeh    Nasser  Alruwayeh      M   
257412      879211.0         Phil Toon      Phil       Toon      M   
257474      489030.0     Nycolas Joyal   Nycolas      Joyal      M   
257584      799012.0     Pedro Martins     Pedro    Martins      M   
257614      261255.0  Claudia Espinosa   Claudia   Espinosa      F   

       countryOfOriginName     regionName  affiliateId     affiliateName  \
257361              Kuwait           Asia       4369.0       CrossFit Q8   
257412       United States  North America       1893.0   CrossFit Naples   
257474              Canada  North America          NaN               NaN   
257584              Brazil  South America      15173.0  CrossFit Barueri   
257614             Ecuador  South America          NaN               NaN   

         age  height  weight  overallRank  overallScore  year competition  \
257361  29.0  183 cm  105 kg         -1.0         552.0  2022  semifinals   
257412  24.0   70 in  195 lb         -1.0         496.0  2022  semifinals   
257474  28.0   70 in  200 lb         -1.0         432.0  2022  semifinals   
257584  32.0  178 cm   90 kg         -1.0         464.0  2022  semifinals   
257614  31.0  175 cm   68 kg         -1.0         444.0  2022  semifinals   

        height_cm  weight_kg  
257361      183.0      105.0  
257412      178.0       89.0  
257474      178.0       91.0  
257584      178.0       90.0  
257614      175.0       68.0

In [20]:
atypical_overall_rank = crossfit_df[crossfit_df["overallRank"] < 0][["competitorId", "competitorName"]]
atypical_overall_rank

competitorId    competitorName
257361      458446.0  Nasser Alruwayeh
257412      879211.0         Phil Toon
257474      489030.0     Nycolas Joyal
257584      799012.0     Pedro Martins
257614      261255.0  Claudia Espinosa

La variable "overallRank" sí presenta valores atípicos. He filtrado y creado una nueva variable para los atletas que tienen una posición negativa en el ranking para saber cuáles son.

Solo son 5 atletas, se podría considerar en un futuro tratar estos datos como nulos para que no distorsionen nuestro análisis.

## 2. Limpieza y transformación de los datos

Voy a eliminar las columnas "height" y "weight", ya que están duplicadas pero con otras unidades de medida. Estas mismas columnas están en kilogramos (weight_kg) y centímetros (height_cm) y me voy a quedar con ellas mejor para este análisis.

In [8]:
crossfit_df.drop(["height", "weight"], axis = 1, inplace=True)

Voy a renombrar algunas columnas para que se entiendan mas facilmente y voy a ponerlas en un formato de nomenclatura genérico.

In [9]:
# Función de regex para añadir una "_" en medio de dos palabras y poner todo en minúscula
import re

crossfit_df.columns = [re.sub(r'([a-z])([A-Z])', r'\1_\2', col).lower() for col in crossfit_df.columns]

In [10]:
crossfit_df = crossfit_df.rename(columns={
    "country_of_origin_name": "country",
    "region_name": "region",
    "overall_rank" : "rank",
    "overall_score" : "score"
})
crossfit_df.columns

Index(['competitor_id', 'competitor_name', 'first_name', 'last_name', 'gender',
       'country', 'region', 'affiliate_id', 'affiliate_name', 'age', 'rank',
       'score', 'year', 'competition', 'height_cm', 'weight_kg'],
      dtype='object')

#### TRATAMIENTO DE VALORES NULOS 

Las variables "affiliate_id" y "affiliate_name" son las que cuentan con mayor porcentaje de valores nulos en mi dataset. 

Estas variables hacen referencia al nombre del sitio donde entrenan estos atletas, lo cuál considero innecesario, ya que, cuento con el país y la región para establecer relaciones y hacer estudios demográficos y como variable de identificación voy a usar los nombres y la variable "competitor_id".

Por tanto voy a eliminar también estas dos variables.

In [11]:
crossfit_df.drop(["affiliate_id", "affiliate_name"], axis = 1, inplace=True)

Centrándonos en la variable "country" voy a sustituir los valores nulos por "NoCountry" para indicar que no disponemos de este valor. Considero que esto es más beneficioso para nuestro posterior análisis demográfico, ya que, tendremos en cuenta también a estos atletas.

In [25]:
crossfit_df["country"] = crossfit_df["country"].fillna("NoCountry")

In [26]:
crossfit_df[crossfit_df["country"] == "NoCountry"]

competitor_id     competitor_name  first_name   last_name gender  \
12           901702.0        Stas Solodov        Stas     Solodov      M   
26          1325298.0     Aleksandar Ilin  Aleksandar        Ilin      M   
49           229227.0     Iakov filimonov       Iakov   filimonov      M   
60           975774.0    Roman Khrennikov       Roman  Khrennikov      M   
103          692539.0      Denis Zalozniy       Denis    Zalozniy      M   
...               ...                 ...         ...         ...    ...   
398827      1638688.0  Anastasia Smirnova   Anastasia    Smirnova      F   
398829       822417.0    Milana Yakovleva      Milana   Yakovleva      F   
398833      1516658.0       Raisa Blinova       Raisa     Blinova      F   
398884       675164.0       Nikita Yundov      Nikita      Yundov      M   
399270       672425.0      Arthur Semenov      Arthur     Semenov      M   

          country  region   age   rank   score  year competition  height_cm  \
12      NoCountry    Asia  36.0   39.0   548.0  2021        open      175.0   
26      NoCountry    Asia  31.0   27.0   748.0  2021        open      175.0   
49      NoCountry    Asia  33.0   51.0  1061.0  2021        open      171.0   
60      NoCountry    Asia  26.0   40.0  1304.0  2021        open      182.0   
103     NoCountry    Asia  34.0  113.0  1988.0  2021        open      162.0   
...           ...     ...   ...    ...     ...   ...         ...        ...   
398827  NoCountry    Asia  30.0    9.0   425.0  2023  semifinals      167.0   
398829  NoCountry    Asia  29.0   11.0   402.0  2023  semifinals      159.0   
398833  NoCountry    Asia  28.0   16.0   366.0  2023  semifinals      172.0   
398884  NoCountry  Europe  30.0   54.0    34.0  2023  semifinals      178.0   
399270  NoCountry    Asia  30.0    NaN     0.0  2023       games      189.0   

        weight_kg  
12           85.0  
26           87.0  
49           82.0  
60           95.0  
103          71.0  
...           ...  
398827       70.0  
398829       58.0  
398833       73.0  
398884       90.0  
399270       99.0  

[2755 rows x 14 columns]

#### TRATAMIENTO DE VALORES ATÍPICOS

Como hemos visto antes para la variable "rank" existían algunos valores atípicos, es decir, negativos para algunos atletas.
Voy a transformar estos valores negativos en valores nulos para que así sigan formando parte del análisis y sean tratados como valores nulos.

In [27]:
crossfit_df.loc[crossfit_df["rank"] < 0, "rank"] = np.nan

In [12]:
rank_nan = crossfit_df[crossfit_df["rank"].isna()]
rank_nan[["competitor_id", "competitor_name", "rank"]]

competitor_id            competitor_name  rank
399262       665223.0          Michal Wesolowski   NaN
399263        81616.0  Björgvin Karl Guðmundsson   NaN
399264       234569.0             David Shorunke   NaN
399265      1762608.0           Morteza Sedaghat   NaN
399266       250172.0       Bronislaw Olenkowicz   NaN
...               ...                        ...   ...
399337       872275.0           Caroline Stanley   NaN
399338       174547.0            Amanda Barnhart   NaN
399339       121033.0               Sydney Wells   NaN
399340       503582.0              Alexis Raptis   NaN
399341      1547449.0                  Emma Cary   NaN

[80 rows x 3 columns]

Tenemos 85 valores nulos (80 valores que ya eran nulos + 5 valores que eran negativos y acabo de transformalos en nulos)

Voy a transformar el tipo de las variables de edad (age) y "competitor_id" para que sean tratados como valores integer.

In [29]:
crossfit_df["age"] = crossfit_df["age"].astype(int)
crossfit_df["competitor_id"] = crossfit_df["competitor_id"].astype(int)
crossfit_df.dtypes

competitor_id        int64
competitor_name     object
first_name          object
last_name           object
gender              object
country             object
region              object
age                  int64
rank               float64
score              float64
year                 int64
competition         object
height_cm          float64
weight_kg          float64
dtype: object

Compruebo que la variable "competitor_id" sólo cuenta con valores numéricos para la identificación de los atletas y que las variables "competitor name" , "country" y "region" sólo cuentan con valores de tipo object.

In [30]:
# Intento convertir la columna a numérica y busco valores no convertibles (NaN)
no_numericos = crossfit_df[pd.to_numeric(crossfit_df["competitor_id"], errors="coerce").isna()]

# Muestro los casos con valores no numéricos (si existen)
print(no_numericos[["competitor_id"]])

Empty DataFrame
Columns: [competitor_id]
Index: []


In [14]:
columns = ["competitor_name", "country", "region"]

# Selecciono filas donde el valor no es de tipo string y muestro los casos (si existen)
for col in columns:
    no_texto = crossfit_df[~crossfit_df[col].apply(lambda x: isinstance(x, str))]
    print(f"Valores no texto en la columna {col}:")
    print(no_texto[[col]])

Valores no texto en la columna competitor_name:
       competitor_name
265221             NaN
328489             NaN
Valores no texto en la columna country:
       country
12         NaN
26         NaN
49         NaN
60         NaN
103        NaN
...        ...
398827     NaN
398829     NaN
398833     NaN
398884     NaN
399270     NaN

[2755 rows x 1 columns]
Valores no texto en la columna region:
Empty DataFrame
Columns: [region]
Index: []


Sí que se cumplen las condiciones mencionadas anteriormente, por tanto, no es necesario tratar ninguna fila en específico.

#### TRATAMIENTO DE VALORES DUPLICADOS

Como hemos visto anteriormente, considero que la columna "competitor_id" debería tener los menores valores duplicados posibles, aunque cuenta con información valiosa en cuanto a si un atleta ha participado en varias competiciones en el mismo año.

In [15]:
crossfit_df.duplicated().sum()

np.int64(0)

Esto nos indica que no hay ninguna fila completamente igual a otra, por tanto, vamos a dejar el análisis con los duplicados encontrados.

#### EXPORTACIÓN DEL NUEVO DATASET (LIMPIO)

In [35]:
crossfit_df.to_csv("D:\\firstprojecteda\\data\\clean_consolidated_athletes_information.csv", index=False)

# Uso index = False para que no se me guarde un nuevo índice automáticamente para cada fila en el nuevo csv